sqlmesh/dbt/adapter.py

```diff
@@ -276,6 +290,20 @@ def execute(
             return AdapterResponse("Success"), pandas_to_agate(resp)
         return AdapterResponse("Success"), empty_table()
 
+    def resolve_schema(self, relation: BaseRelation) -> t.Optional[str]:
+        schema = self._map_table_name(relation.database, relation.schema, relation.identifier).db
+        if not schema:
+            return None
```

Comment:
Is it valid to get None back? If not, should we raise instead? Same with resolve_identifier.

Reply:
It is. It matches the behavior of the relation itself.